diff --git a/dist/gtk4/jsx-runtime.ts b/dist/gtk4/jsx-runtime.ts
index 6d7c863ee3e8ca73078451d100172ff58a1df99e..937e5df4c98000022fde15b3fd368665c3bfdf49 100644
--- a/dist/gtk4/jsx-runtime.ts
+++ b/dist/gtk4/jsx-runtime.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import Gtk from "gi://Gtk?version=4.0"
 import Gio from "gi://Gio?version=2.0"
 import { configue } from "../jsx/env.js"
diff --git a/dist/jsx/state.ts b/dist/jsx/state.ts
index 0c5cc6eb153375e7bed635e6364ee024b8b06eca..dc856f0d53968266ee0e5fbe2787bec815985eac 100644
--- a/dist/jsx/state.ts
+++ b/dist/jsx/state.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 /* eslint-disable @typescript-eslint/no-unsafe-declaration-merging */
 
 import GObject from "gi://GObject"
